### PR TITLE
Adds eval, lags, corr and season commands to the econometrics menu

### PR DESCRIPTION
**Pull request by northern-64bit**
# Description

Adds eval (from the economy menu), lag (a new command), corr and season (from the forecast menu) commands to the econometrics menu.

# How has this been tested?

Running the commands and getting the same/expected results as run in the other menus.
- [x] Make sure affected commands still run in terminal
- [x] Ensure the SDK still works
- [x] Check any related reports

# Checklist:

- [x] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
- [x] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/econometrics/econometrics_controller.py [hunk 1]
```diff
@@ -282,6 +293,329 @@ def print_help(self):
...
+        batch_size: Optional[int] = None,
+        learning_rate: bool = False,
+        past_covariates: bool = False,
+        all_past_covariates: bool = False,
+        lags: bool = False,
+        hidden_size: int = 0,
+        n_jumps: bool = False,
+        end: bool = False,
+        start: bool = False,
+        residuals: bool = False,
```
Comment (jmaslek) [inline openbb_terminal/econometrics/econometrics_controller.py:324]: These are all ML things and shouldnt be required here

**Comment by jmaslek** [comment]
Unlabeled seasonality plot 
Screenshot 2023-03-17 at 9 11 58 AM

**Comment by jmaslek** [comment]
`lag` does not create a new variable: it overwrtites the current one

Screenshot 2023-03-17 at 9 16 33 AM

**Comment by northern-64bit** [comment]
@jmaslek All issues should now have been fixed, except for labels on the plot of the `season` command. 

This hasn't been done since this is a general issue (which also impacts the `season` command of the `forecast menu`) with `add_corr_plot` of `plotly_helper`. I can't see a good solution for this due to each line is added as it's own scatter. Therefore I'll create a new issue for it.

**Review by jmaslek — APPROVED** [review]
Nice one!
